LICENSE

```diff
@@ -1,22 +1,213 @@
-MIT License
+[airpyllution] Copyright (2022) (Christopher Alexander, Daniel King, Mel Liow)(“Licensor”)
```

Comment:
I think it should be `airpyllution Copyright 2022 Christopher Alexander, Daniel King, Mel Liow` Not sure whether to include (“Licensor”) or not

Reply:
Done